iree/compiler/Dialect/HAL/Target/VulkanSPIRV/test/BUILD

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
Nit: don't update copyright years for moved files

Reply:
I created a new file and deleted the old one, but git thinks it was a "move" ;P

Reply:
Ah, well I'd just switch this back to 2019 and call it a move, but :shrug: